.github/workflows/checks.yml

```diff
@@ -16,6 +16,5 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 12
-      - run: npm install --global gulp-cli
       - run: yarn --version
       - run: yarn cichecks
```

Comment:
hmmm, will removing npm or yarn install not break line 20? Do we need step yarn run build:prod also?